datahub-web-react/src/app/entity/dataset/profile/schema/components/SchemaHeader.tsx

```diff
@@ -57,21 +149,54 @@ export default function SchemaHeader({
     hasKeySchema,
     showKeySchema,
     setShowKeySchema,
+    lastUpdatedTimeString,
+    schemaBlameSelectValue,
+    semVerList,
```

Comment:
i know this is maybe a bit annoying - but where possible we should write out full names instead of abbreviations.. ie: semanticVersion

Reply:
Done!